public_html/administrator/components/com_jed/views/review/tmpl/edit.php

```diff
@@ -0,0 +1,159 @@
+            </a></td>
+    </tr>
+    <tr>
+        <td><?php echo Text::_('COM_JED_EXTENSIONS_DEVELOPER'); ?></td>
+        <td><a href="<?php echo 'index.php?option=com_users&task=user.edit&id=' . (int) $this->item->developer_id; ?>" title="<?php echo $this->escape($this->item->developer); ?>">
+		        <?php echo $this->escape($this->item->developer); ?>
+            </a></td>
+    </tr>
+    <tr>
+        <td><?php echo Text::_('COM_JED_REVIEWS_VERSION'); ?></td>
+        <td><?php echo $this->item->version; ?></td>
+    </tr>
+    <tr>
+        <td><?php echo Text::_('COM_JED_REVIEWS_SCORE_FUNCTIONALITY'); ?></td>
+        <td><?php echo $this->item->functionality; ?></td>
+    </tr>
+    <tr>
+        <td><?php echo Text::_('COM_JED_REVIEWS_COMMENTS_FUNCTIONALITY'); ?></td>
+        <td><?php echo $this->item->functionalityComment; ?></td>
+    </tr>
+    <tr>
+        <td><?php echo Text::_('COM_JED_REVIEWS_SCORE_EASE_OF_USE'); ?></td>
+        <td><?php echo $this->item->easeOfUse; ?></td>
+    </tr>
+    <tr>
+        <td><?php echo Text::_('COM_JED_REVIEWS_COMMENTS_EASE_OF_USE'); ?></td>
+        <td><?php echo $this->item->easeOfUseComment; ?></td>
+    </tr>
+    <tr>
+        <td><?php echo Text::_('COM_JED_REVIEWS_SCORE_SUPPORT'); ?></td>
+        <td><?php echo $this->item->support; ?></td>
+    </tr>
+    <tr>
+        <td><?php echo Text::_('COM_JED_REVIEWS_COMMENTS_SUPPORT'); ?></td>
+        <td><?php echo $this->item->supportComment; ?></td>
+    </tr>
+    <tr>
+        <td><?php echo Text::_('COM_JED_REVIEWS_SCORE_DOCUMENTATION'); ?></td>
+        <td><?php echo $this->item->documentation; ?></td>
+    </tr>
+    <tr>
+        <td><?php echo Text::_('COM_JED_REVIEWS_COMMENTS_DOCUMENTATION'); ?></td>
+        <td><?php echo $this->item->documentationComment; ?></td>
+    </tr>
+    <tr>
+        <td><?php echo Text::_('COM_JED_REVIEWS_SCORE_VALUE_FOR_MONEY'); ?></td>
+        <td><?php echo $this->item->valueForMoney; ?></td>
+    </tr>
+    <tr>
+        <td><?php echo Text::_('COM_JED_REVIEWS_COMMENTS_VALUE_FOR_MONEY'); ?></td>
+        <td><?php echo $this->item->valueForMoneyComment; ?></td>
+    </tr>
+
+    <tr>
+        <td><?php echo Text::_('COM_JED_REVIEWS_URL'); ?></td>
+        <td><?php echo $this->item->url; ?></td>
+    </tr>
+    <tr>
+        <td><?php echo Text::_('COM_JED_REVIEWS_USED_FOR'); ?></td>
+        <td><?php echo $this->item->usedFor; ?></td>
+    </tr>
+
+    <tr>
+        <td><?php echo Text::_('COM_JED_REVIEWS_IP_ADDRESS'); ?></td>
+        <td><?php echo $this->item->ipAddress; ?></td>
+    </tr>
+    <tr>
+        <td><?php echo Text::_('COM_JED_REVIEWS_FLAGGED'); ?></td>
+        <td>
+			<?php
+			echo (int) $this->item->flagged === 1 && !is_null($item->ipAddress)
+				? HTMLHelper::_(
+					'link',
+					'https://batchrev.extensions.joomla.org/ipaddress/' . $this->item->ipAddress . '.html',
+					Text::_('JYES'),
+					'target="_blank"'
```

Comment:
This is not done huh in Joomla ;) We do show an icon telling the user this is going to open in a new tab
```
<span class="icon-new-tab"></span>
```